src/index.jsx

```diff
@@ -4,32 +4,33 @@ import 'regenerator-runtime/runtime';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import {
-  Route,
-  Switch,
-} from 'react-router-dom';
-import { Provider } from 'react-redux';
-import { ConnectedRouter } from 'react-router-redux';
-import { Helmet } from 'react-helmet';
-import { PrivateRoute } from '@edx/frontend-auth';
-import * as FullStory from '@fullstory/browser';
+  initialize, APP_INIT_ERROR, APP_READY, subscribe,
+} from '@edx/frontend-platform';
+import { ErrorPage } from '@edx/frontend-platform/react';
 
-import EnterpriseApp from './containers/EnterpriseApp';
-import Header from './containers/Header';
-import Footer from './containers/Footer';
-import EnterpriseIndexPage from './containers/EnterpriseIndexPage';
-import AdminRegisterPage from './containers/AdminRegisterPage';
-import UserActivationPage from './containers/UserActivationPage';
-import JwtAuthenticationRefresh from './components/JwtAuthenticationRefresh';
-import NotFoundPage from './components/NotFoundPage';
-import SupportPage from './components/SupportPage';
-import { withErrorBoundary } from './components/ErrorBoundary';
-import { ToastsProvider, Toasts } from './components/Toasts';
+import * as FullStory from '@fullstory/browser';
 
-import apiClient from './data/apiClient';
-import store from './data/store';
-import history from './data/history';
 import { configuration } from './config';
 
+import App from './components/App';
+
+subscribe(APP_READY, () => {
+  ReactDOM.render(<App />, document.getElementById('root'));
+});
+
+subscribe(APP_INIT_ERROR, (error) => {
+  ReactDOM.render(<ErrorPage message={error.message} />, document.getElementById('root'));
+});
+
+initialize({
+  messages: [],
+  // We don't require authenticated users so that we can perform our own auth redirect to a proxy login that depends on
+  // the route, rather than the LMS like frontend-platform does.
```

Comment:
nit: I'm guessing this might be copy/paste from the Learner Portal but Admin Portal doesn't have use its own auth redirect to proxy login quite yet (filed ENT-4040 to cover this).

Should we just remove this comment for now?